src/bindings/js/node/src/tensor.cpp

```diff
@@ -184,6 +186,19 @@ Napi::Value TensorWrap::set_shape(const Napi::CallbackInfo& info) {
     return info.Env().Undefined();
 }
 
+Napi::Value TensorWrap::copy_to(const Napi::CallbackInfo& info) {
```

Comment:
copy_to should be void similarily to https://github.com/openvinotoolkit/openvino/blob/133852e6e216967de383242422f6ff6cbf7bdf33/src/bindings/js/node/src/core_wrap.cpp#L448